xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -1433,6 +1425,29 @@ void CXBMCApp::onReceive(CJNIIntent intent)
   }
 }
 
+void CXBMCApp::OnSleep()
+{
+  CLog::Log(LOGINFO, "CXBMCApp::OnSleep");
```

Comment:
```suggestion
  CLog::Log(LOGDEBUG, "CXBMCApp::OnSleep");
```
Another similar log line already exists: "Got device sleep intent"